api/contrib/envoy/extensions/http/cluster_specifier/golang/v3alpha/golang.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package envoy.extensions.http.cluster_specifier.golang.v3alpha;
+
+import "google/protobuf/any.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.http.cluster_specifier.golang.v3alpha";
+option java_outer_classname = "GolangProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/http/cluster_specifier/golang/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Golang]
+//
+// For an overview of the Golang cluster specifier please see the :ref:`configuration reference documentation <config_http_cluster_specifier_golang>`.
+// [#extension: envoy.router.cluster_specifier_plugin.golang]
+
+// [#extension-category: envoy.router.cluster_specifier_plugin]
+message Config {
+  // Globally unique ID for a dynamic library file.
+  string library_id = 1 [(validate.rules).string = {min_len: 1}];
```

Comment:
We can separate it into a common type, which can be reused with the Go extension API. such as
```
message Library {
string id = 1 [(validate.rules).string = {min_len: 1}];
string path = 2 [(validate.rules).string = {min_len: 1}];
}
```

Reply:
I prefer to keep it simple now since we could only reuse two fields, maybe we could turn to reuse it when we move dso into bootstrap.